packages/security/src/components/Header/Header-story.js

```diff
@@ -40,15 +44,15 @@ disableCenteredStories(storiesOf(patterns('Header#legacy'), module))
             rel="noopener noreferrer"
             target="_blank"
           >
-            More info
+            View replacement
           </NotificationActionButton>
         }
         kind="info"
-        subtitle="This component is now legacy. Please click to see the go-forward component"
+        subtitle="Pattern no longer supported. The pattern will remain available, but plan to migrate to the pattern replacement."
         title=""
         hideCloseButton
+        style={{ display: 'block' }}
```

Comment:
Are we intentionally breaking this one out into a separate line?
![image](https://user-images.githubusercontent.com/1717398/167485920-80bc8872-34a1-4979-97b1-56b6534c03d0.png)